frontend/src/components/MapView/Layers/layer-utils.tsx

```diff
@@ -120,22 +143,28 @@ export const addPopupParams = (
     });
   }
 
+  // temporary fix for the admin level
+  const possibleAdminLevelData: PopupData = adminLevel
+    ? {
+        'Admin Level': {
+          data: feature.properties.adminLevel,
+          coordinates,
+        },
+      }
+    : {};
+
+  const featureInfoPropsData = getFeatureInfoPropsData(
+    featureInfoPropsWithFallback || {},
+    coordinates,
+    feature,
+  );
+
   dispatch(
     addPopupData({
-      // temporary fix for the admin level
-      ...(adminLevel
-        ? {
-            'Admin Level': {
-              data: feature.properties.adminLevel,
-              coordinates,
-            },
-          }
-        : {}),
-      ...getFeatureInfoPropsData(
-        featureInfoPropsWithFallback || {},
-        coordinates,
-        feature,
-      ),
+      // Only if we're providing a custom label, put the data before admin level
+      ...(!useCustomLabel ? possibleAdminLevelData : {}),
+      ...featureInfoPropsData,
+      ...(useCustomLabel ? possibleAdminLevelData : {}),
```

Comment:
Is that change expected for CH where the adminLevel ends up at the bottom?

Before:
<img width="557" alt="Screenshot 2024-06-27 at 3 01 56 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/047c5d9d-aecd-4130-8e70-9ede1ce9dabd">

After:
<img width="587" alt="Screenshot 2024-06-27 at 3 02 01 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/2be20888-c169-43cc-bcac-d2b1911b6c53">

Reply:
Good catch, I wouldn't consider this expected. I'm not certain that I know where we'd prefer "Admin Level" to go in the tooltip order.

@wadhwamatic, for tooltips showing the "IFPRI / ACLED Conflict Analysis - 2023" layer, do you have a preference for where the Admin Level is displayed?

> **Mali, Tombouctou, Tombouctou**
> IFPRI / ACLED Conflict Analysis - 2023
> _Admin Level: 2 (this is where it was prior to this change)_
> Classification: High
> Deadliness score: 84
> Deadliness rank: 47
> Danger score: 34
> Danger rank: 16
> Diffusion score: 2
> Diffusion rank: 65
> Fragmentation score: 4
> Fragmentation rank: 6
> _Admin Level: 2 (this is where's it's currently with this change)_

Reply:
@gislawill - we can keep admin level at the bottom. In the issue I'll write up today, we should also address the need for this. It's not always necessary to display it.